homeassistant/components/flipr/config_flow.py

```diff
@@ -48,79 +51,23 @@ async def async_step_user(
                 errors["base"] = "unknown"
                 _LOGGER.exception("Unexpected exception")
 
-            if not errors and not flipr_ids:
-                # No flipr_id found. Tell the user with an error message.
-                errors["base"] = "no_flipr_id_found"
+            else:
+                _LOGGER.debug("Found flipr or hub ids : %s", ids)
 
-            if errors:
-                return self._show_setup_form(errors)
+                if len(ids["flipr"]) > 0 or len(ids["hub"]) > 0:
+                    # If there is a flipr or hub, we can create a config entry.
+                    # No unique id for config entry to set. Using the one generated by home assistant.
```

Comment:
Do we have something unique from the API like an user id? Can you change the emailadress?

Reply:
The emailadress can not be changed.
There is a user_id in the cloud api. Getting it requires to add a implement a simple call for a method to be created 'get_user_id'.
What do you prefer I put as unique_id ?

Reply:
I put email in the correction.